### PR TITLE
Feature/72

### LINKED ISSUE #72: Experiment Card - Design the delete experiment dialog
Please refer to the screenshot
Screen Shot 2022-03-25 at 12 41 12 PM

### PR DESCRIPTION
**Pull request by paularah**
Closes #72 

Implemented solution: ... 

adds delete experiment dialog and  a backdrop for the delete action 

How to test this PR: ...

### Sanity checks:
- [x] The pull request is explicitly linked to the relevant issue(s)
- [x] The issue is well described: clearly states the problem and the general proposed solution(s)
- [x] From the issue and the current PR it is explicitly stated how to test the current change
- [x] The labels in the issue set the scope and the type of issue (bug, feature, etc.)
- [x] All the automated test checks are passing
- [ ] All the linked issues are included in one milestone
- [x] All the linked issues are assigned

### Breaking changes (select one):
- [x] The present changes do not change the preexisting api in any way
- [ ] This PR and the issue are tagged as a `breaking-change`

### Possible deployment updates issues (select one):
- [x] There is no reason why deployments may break after the current update
- [ ] This PR and the issue are tagged as `alert:deployment`

### Test coverage (select one):
- [ ] Tests for the relevant cases are included in this pr
- [x] The changes included in this pr are out of the current test coverage scope

### Documentation (select one):
- [ ] The documentation has been updated to match the current changes
- [x] The changes included in this PR are out of the current documentation scope

### Nice to have (if relevant):
Screenshot from 2022-03-29 12-54-53

- [ ] Screenshots of the changes
- [ ] Explanatory video/animated gif

### CODE REVIEW

File: applications/salk-portal/package-lock.json [hunk 1]
```diff
@@ -1812,6 +1812,24 @@
         "@turf/helpers": "^6.5.0"
       }
     },
```
Comment (vidhya-metacell) [inline applications/salk-portal/package-lock.json:1814]: @paularah please remove this file, do not push this, keep only on your local env

File: applications/salk-portal/src/components/home/DeleteExperimentDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      open={open}
+      onClose={handleClose}
+    >
+      <DialogTitle>
+      Delete this experiment?
+        <IconButton onClick={handleClose}>
+          <img src={CLOSE} alt="close" />
+        </IconButton>
+      </DialogTitle>
+      <DialogContent dividers={false} className={classes.info}>
```
Comment (paularah) [inline applications/salk-portal/src/components/home/DeleteExperimentDialog.tsx:43]: @vidhya-metacell

Comment (vidhya-metacell) [reply applications/salk-portal/src/components/home/DeleteExperimentDialog.tsx:43]: @paularah the ones you see on the UI are not the dividers they are instead boxshadows

File: applications/salk-portal/src/components/home/DeleteExperimentDialog.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+interface IDeleteDialog {
+    open:boolean;
+    handleClose: () => void
+}
+
+export const DeleteExperimentDialog: React.FC<IDeleteDialog> = ({open, handleClose}) => {
+  const classes = useStyles()
+
+  return (
```
Comment (vidhya-metacell) [inline applications/salk-portal/src/components/home/DeleteExperimentDialog.tsx:30]: @paularah you can reuse the already existing Basedialog, i know it has slight different UI like the button color, shadows and padding, that can be easily handles with a class

File: applications/salk-portal/src/components/home/DeleteExperimentDialog.tsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    <Dialog
+      fullWidth={true}
+      maxWidth='xs'
+      open={open}
+      onClose={handleClose}
+    >
+      <DialogTitle>
```
Comment (vidhya-metacell) [inline applications/salk-portal/src/components/home/DeleteExperimentDialog.tsx:37]: @paularah when you inspect you would notice MuiDialogTitle-root - font-weight, line-height, color is not correct
MuiDialogContent-root - Letter spacing issue, color not added

Comment (vidhya-metacell) [reply applications/salk-portal/src/components/home/DeleteExperimentDialog.tsx:37]: @paularah you could even message Paul Faivret and ask him if we can keep similar styles for both the modals

**Pull request closed without merging**